### PR TITLE
Range highlights should reflect options.reversed when set

### PR DESCRIPTION
**Pull request by gormanb**
Pull Requests
=============
Please accompany all pull requests with the following (where appropriate):

- [x] unit tests (we use [Jasmine 1.3](http://jasmine.github.io/1.3/introduction.html))
- [ ] JSFiddle (or an equivalent such as CodePen, Plunker, etc) with new feature or bug-fix
- [ ] Link to original Github issue (if this is a bug-fix)
- [ ] documentation updates to README file
- [ ] examples within [/tpl/index.tpl](https://github.com/seiyria/bootstrap-slider/blob/master/tpl/index.tpl) (for new options being added)
- [x] Passes CI-server checks (runs automated tests, JS source-code linting, etc..). To run these on your local machine, type `grunt test` in your Terminal within the bootstrap-slider repository directory

### CODE REVIEW

**Comment by seiyria** [comment]
Thanks! Does this resolve any particular issues that you can reference?

File: test/specs/RangeHighlightsSpec.js [hunk 1]
```diff
@@ -38,6 +40,39 @@ describe("RangeHighlights Render Tests", function() {
...
+            rangeHighlights: [
+                { "start": 2, "end": 5 },    // left: 75%; width: 15%
+                { "start": 7, "end": 8 },    // left: 60%; width: 5%
+                { "start": 17, "end": 19 },  // left: 5%; width: 10%
+                { "start": 17, "end": 24 },  //out of range - not visible
+                { "start": -3, "end": 19 }   //out of range - not visible
+            ]
+        });
+
+        testSlider4 = $('#testSlider4').slider({
```
Comment (rovolution) [inline test/specs/RangeHighlightsSpec.js]: could you properly tear down `testSlider4` in the `afterEach` block (lines 79 - 85)?

File: test/specs/RangeHighlightsSpec.js [hunk 2]
```diff
@@ -38,6 +40,39 @@ describe("RangeHighlights Render Tests", function() {
                 { "start": 23, "end": 15 }  //out of range - not visible
             ]
         });
+
+        testSlider3 = $('#testSlider3').slider({
```
Comment (rovolution) [inline test/specs/RangeHighlightsSpec.js]: could you properly tear down `testSlider3` in the `afterEach` block (lines 79 - 85)?

**Comment by gormanb** [comment]
Hey @seiyria - no, having looked through the open issues there don't appear to be any which refer to this behaviour.

@rovolution: I've made the additional code changes as requested. Below is a series of comparison images showing the range-highlighting before and after the change, with `reversed` set to `false` and `true` in turn. The slider runs from 0 to 100 and the highlighted range is from 24 to 25 in each case. All images are captured with the slider handle at 0.

bootstrap-slider-highlight-resize

**Comment by rovolution** [comment]
Published to 9.2.2 - thanks for the PR @gormanb!

**Pull request merged**